index.html

```diff
@@ -3613,6 +3613,27 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Contributor's card END ________  -->
 
+<!-- ________ bdwolff's card START ________  -->
```

Comment:
Your card content is good but it all needs to be cut and pasted above the line with the following comment:
`<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->`
right now your card is at the bottom of the file, and it should be above the template card.